src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -262,7 +262,7 @@ def create(self, daemon_spec: CephadmDaemonSpec) -> str:
         # get mon. key
         ret, keyring, err = self.mgr.check_mon_command({
             'prefix': 'auth get',
-            'entity': 'mon.',
+            'entity': self.get_auth_entity(name),
```

Comment:
not sure here. Are you really sure we want to change this?

Reply:
#36936 disallows removal of the mon keyring (https://github.com/ceph/ceph/pull/36936/commits/3589975ab2cd10e9f3197685376803cad6dc1294)

is there something else we should be concerned with?

Reply:
1. Simply doing this here would introduce an inconsistency as the first MON is deployed using the `mon.` keyring. 
2. https://docs.ceph.com/docs/master/rados/configuration/auth-config-ref/#enabling-cephx recommends to use a single keyring for the MON cluster. 

So. :-1: for simply changing this here. Changing this would IMO deserve it's own PR.

Reply:
this is hardcoded in `get_auth_entity` to always return `mon.` for a this type ..

we could assert for this?
```
assert self.get_auth_entity(name) == 'mon.' 
```

.. but the `auth get` is going fail anyway when the entity does not match what was created at bootstrap ..